verification/verifier_helpers.go

```diff
@@ -37,11 +160,11 @@ func verifyX509TrustedIdentities(certs []*x509.Certificate, trustPolicy *TrustPo
 		return nil
 	}
```

Comment:
We should return an error in this case as there is no x509 subject trusted identity to consider.